addons/website_event_track/views/event_track_templates_list.xml

```diff
@@ -300,71 +300,82 @@
 <!-- TOOL TEMPLATES -->
 <!-- ============================================================ -->
 
-<template id="tracks_cards_track" name="Track Card">
```

Comment:
Why so much changes ? Seems a lot of diff, not sure why as changes are nearly impossible to spot. Previously we had a "per card" template, seems you introduced a multi-card, not sure why.